.github/workflows/ossf-scorecard.yml

```diff
@@ -4,24 +4,26 @@ name: OSSF Scorecard
 on:
   # For Branch-Protection check. Only the default branch is supported. See https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection.
   branch_protection_rule:
-  push:
-    branches: [main]
   # To guarantee Maintained check is occasionally updated. See https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained.
   schedule:
     - cron: "0 0 * * 1"
+  push:
+    branches: [main, master]
   workflow_dispatch:
 # Declare default permissions as read only.
 permissions: read-all
 jobs:
   analysis:
     name: Scorecard analysis
     runs-on: ubuntu-24.04
+    # Delete the conditional below if you are using the OSSF Scorecard on a private repository.
+    if: ${{ github.event.repository.private == false }}
```

Comment:
```suggestion
```
This will be a public repo so we don't need this check either -- side note: I just realized I need to fix the comment.